### DISCUSSION TITLE
Warning when renaming Entity ID

### DISCUSSION THREAD

**Discussion by tuxick** [home-assistant/frontend#8386] [Configuration]
Dialogs allow renaming entity_id but they don't warn that means losing them in cards, automations and history.

**Comment by Masterz69** [comment]
IMHO it clearly describes what will happen:
"
This will not change any configuration (like automations, scripts, scenes, dashboards) that is currently using these entities! 
You will have to update them yourself to use the new entity IDs!
"
.

**Reply by spacegaier** [reply]
We I had added that 5 months ago to warn users.
@tuxick Which HA version are you running?
